### DISCUSSION TITLE
Better audio speed

### DISCUSSION THREAD

**Discussion by ruippgoncalves** [ThioJoe/Auto-Synced-Translated-Dubs#60] [General]
The youtube playback speed uses the default browser, so in your case, it is Chrome, you can look at chromium (the open source part of the browser) and see what they are using... 

https://www.chromium.org/developers/design-documents/video/

But corting short, they are using ffmpeg which is open source and can be used to improve the audio quality.

**Comment by ThioJoe** [comment]
Hmm interesting

**Comment by jn-jairo** [comment]
I believe you are looking for the ffmpeg option [atempo](https://www.ffmpeg.org/ffmpeg-filters.html#atempo).

It does a great job, but I use [SoX](https://sox.sourceforge.net) to change audio duration without changing pitch and it has the same quality as ffmpeg if you use the right option.

I use the command line but there is a python library [pysox](https://pypi.org/project/sox), you need to use the [tempo](https://pysox.readthedocs.io/en/latest/api.html#sox.transform.Transformer.tempo) method with the option for speech.

**Comment by ThioJoe** [comment]
Moving this to a discussion
